pkgs/development/python-modules/prometheus-pandas/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, ipython
+, matplotlib
+, numpy
+, pandas
+, requests
+}:
+
+buildPythonPackage rec {
+    pname = "prometheus-pandas";
+    version = "0.3.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-zysdlYPjvhextAfW4vvGN6mSPVUbWLV+AvQ1hGWCAbw=";
+    };
+
+    propagatedBuildInputs = [
+      ipython
+      matplotlib
+      numpy
+      pandas
+      requests
+    ];
+    # Tests never work. :(
+    doCheck = false;
+
+    meta = with lib; {
+      homepage = "https://github.com/dcoles/prometheus-pandas";
+      license = licenses.mit;
+      description = "Pandas integration for Prometheus.";
+      # makefu is not the real maintainer
+      maintainers = with maintainers; [viktornordling];
```

Don't know what the purpose of this comment was, but I think it's better to remove it.

```suggestion
      maintainers = with maintainers; [ viktornordling ];
```

---

Ha yeah sorry, this was a leftover from copy pasta. Literally had the removal of that line in my git diff, but hadn't pushed it. 🤦  Cleaning it up!